### DISCUSSION TITLE
Not got the GitHub education benefits even after 5 days

### DISCUSSION THREAD

**Discussion by samip-dash** [community/community#129018] [GitHub Education]
### Hi everyone,

I had submitted a form for GitHub education on June 13 and it was approved and today's it is already June 19, can anybody tell that when will I get my benefits? 
![Screenshot_2024-06-19-06-23-53-74_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/community/community/assets/110279138/13321b15-76d9-4780-85d2-62a48fb541c1)

**Comment by davevad93** [comment]
Hi @samip-dash ,

When the benefits are fully available the state of your application will change to: 

`Congrats! Your request for @samip-dash  was approved and your benefits are now available`

It may take a few days, so be patient. 👍
